soc/v2/raspberrypi/rp2/Kconfig.series

```diff
@@ -5,7 +5,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 config SOC_SERIES_RP2XXX
```

Comment:
The SOC_SERIES_RP2XXX must be defined in Kconfig.soc for reuability, and then Zephyr specific selects stays in the Kconfig.series file.

Try to align with what is done for nRF SoC series: https://github.com/zephyrproject-rtos/zephyr/pull/66648/commits/f78433a4c9f00bf847df6deae334d0eeaaaca0ff